index.html

```diff
@@ -311,72 +330,163 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
+                <a>options list</a> set to a list containing <var>options</var>
+                as its only item.</li>
+              </ol>
+            </li>
+            <li>If <var>options</var>'s <a>buffered</a> flag is set:
+              <ol>
+                <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
+                object returned by the <a href="#filter-buffer-by-name-and-type">
+                </a> algorithm with <var>buffer</var> set to <a>relevant
+                performance entry buffer</a>, <var>name</var> set to
+                <code>null</code> and <var>type</var> set to
+                <var>options</var>'s <a>type</a>.
+                </li>
+                <li>Append <var>entries</var> to the <a data-cite=
+                "WHATWG-DOM#context-object">context object</a>'s <a>observer
+                buffer</a>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          </li>
         </ol>
+        <p class="note">A <a>PerformanceObserver</a> object needs to always call
+        <a>observe()</a> with <var>options</var>'s
+        <a data-link-for=PerformanceObserverInit>entryTypes</a> set OR always
+        call <a>observe()</a> with <var>options</var>'s
+        <a data-link-for=PerformanceObserverInit>type</a> set. This is meant to
+        avoid confusion with how calls would stack. When using
+        <a data-link-for=PerformanceObserverInit>entryTypes</a>, no other
+        parameters in <a>PerformanceObserverInit</a> can be used. In addition,
+        multiple <a>observe()</a> calls will override for backwards
+        compatibility and because a single call should suffice in this case. On
+        the other hand, when using
+        <a data-link-for=PerformanceObserverInit>type</a>, calls will stack
+        because a single call can only specify one type.</p>
         <section data-dfn-for="PerformanceObserverInit" data-link-for=
         "PerformanceObserverInit">
           <h2><dfn>PerformanceObserverInit</dfn> dictionary</h2>
           <pre class="idl">
           dictionary PerformanceObserverInit {
-            required sequence&lt;DOMString&gt; entryTypes;
-            boolean buffered = false;
+            sequence&lt;DOMString&gt; entryTypes;
+            DOMString type;
+            boolean buffered;
           };
           </pre>
           <dl>
             <dt><dfn>entryTypes</dfn></dt>
-            <dd>A list of entry types to be observed. The list MUST NOT be
-            empty and types not recognized by the user agent MUST be
-            ignored.</dd>
+            <dd>A list of entry types to be observed. If present, the list MUST
+            NOT be empty and all other members MUST NOT be present. Types not
+            recognized by the user agent MUST be ignored.</dd>
+          </dl>
+          <dl>
+            <dt><dfn>type</dfn></dt>
+            <dd>A single entry type to be observed. If present, the type MUST be
```

Comment:
What do you mean by "MUST be recognized" here? If the type is unknown, the UA MUST ignore it, no?

Reply:
Done